### PR TITLE
ENH: Move Gifti Mesh IO into a separate module

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
> Looks good to me. Would it make sense, now that Python tests have been moved directly inside each module folder, to add a Python test to make sure that the wrapping works?

Yes! Added.

**Comment by fbudin69500** [comment]
@thewtex I don't see them, but it could just be my eyes...

File: Modules/IO/MeshGifti/test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      ${ITK_TEST_OUTPUT_DIR}/white.gii
+)
+itk_add_test(NAME itkMeshFileReadWriteTestGifti2
+      COMMAND ITKIOMeshGiftiTestDriver itkMeshFileReadWriteTest
+      DATA{Baseline/aparc.gii}
+      ${ITK_TEST_OUTPUT_DIR}/aparc.gii
+)
+
+itk_python_expression_add_test(NAME itkGiftiMeshIOPythonTest
+  EXPRESSION "io = itk.GiftiMeshIO.New()")
```
Comment (dzenanz) [inline Modules/IO/MeshGifti/test/CMakeLists.txt]: @fbudin69500 this is the most basic Python test.

Comment (fbudin69500) [reply Modules/IO/MeshGifti/test/CMakeLists.txt]: @dzenanz : Thanks for pointing out where the test is. For some reason I am pretty sure this was not visible in my browser earlier. Maybe I just needed to refresh my page. Looks good enough to test that the wrapping was successful.

**Pull request merged**